cmd/concourse/generate_key.go

```diff
@@ -74,3 +78,27 @@ func (cmd *GenerateKeyCommand) Execute(args []string) error {
 
 	return nil
 }
+
+type ExtractInternalConfigCommand struct {
```

Comment:
Feels weird having this sub-command in this file. Would make more sense to have it in its own file. wdyt?